python/dstat-kudu/kudu_dstat.py

```diff
@@ -25,23 +21,27 @@ def open_or_create_table(client, table, drop=False):
 
   if not exists:
     # Create the schema for the table, basically all float cols
-    cols = [kudu.ColumnSchema.create("ts", kudu.INT64)]
-    cols += [kudu.ColumnSchema.create(x, kudu.FLOAT) for x in DSTAT_COL_NAMES]
+    builder = kudu.schema_builder()
+    builder.add_column("ts", kudu.int64, nullable=False, primary_key=True)
+    [builder.add_column(col, kudu.float_) for col in DSTAT_COL_NAMES]
```

Comment:
I don't write a lot of python, but this style seems a bit ugly to me, since you're constructing an array of "Nones" for the purpose of its side effect. Maybe a normal for loop would be easier to read?